src/routes/views/components/dataToolbar/dataToolbar.tsx

```diff
@@ -866,7 +866,7 @@ export class DataToolbarBase extends React.Component<DataToolbarProps> {
     // Todo: clearAllFilters workaround https://github.com/patternfly/patternfly-react/issues/4222
     return (
       <div style={style ? style : styles.toolbarContainer}>
-        <Toolbar id="details-toolbar" clearAllFilters={this.onDelete as any} collapseListedFiltersBreakpoint="xl">
+        <Toolbar clearAllFilters={this.onDelete as any} collapseListedFiltersBreakpoint="xl">
```

Comment:
Why has the ID been removed?

The PatternFly examples show an ID for the Toolbar component.
http://patternfly-react-main.surge.sh/components/toolbar#examples

Reply:
There was nothing referencing it, and it was being used twice within a page